src/domUpdates.js

```diff
@@ -0,0 +1,77 @@
+import $ from 'jquery';
+
+export const domUpdates = {
+ displayUserInformation(name, totalAmount) {
+   console.log(name)
+   $('.users-name-display').text(`Hello! ${name}!`)
+   $('.users-total-display').text(`Total Amount Spent: $${totalAmount}.00`)
+ },
+
+ displayManagerInfo(tripsData, numberOnTrips, totalAmount) {
+   $('.agent-name-display').text(`Welcome back: Trond Makonese!`)
+   $('.all-total-display').text(`Total Revenue This Year: $${numberOnTrips}`)
+   $('.users-on-trips-today').text(`Number of Travelers on Trips Today: ${totalAmount}`)
+ },
+
+ displayUsersPastAndPresent(travelHistory) {
+   return travelHistory.forEach(trip => {
+     $('.past-and-present').append(
+       `<div class='all-trips'>
+         <p>Destination: ${trip.destination}</p>
+         <p>Flight Cost Per Person: $${trip.flightPerPerson}.00</p>
+         <p>Estimated Cost For Lodge: $${trip.costPerPersonADay}.00</p>
+         <p>Number of Travelers: ${trip.numberOfTravelers}</p>
+         <p>Date: ${trip.date}</p>
+       </div>
+       `)
+   })
+ },
+
+ displayUserTripInfo(foundInfo) {
+   console.log(foundInfo)
+   return foundInfo.forEach(trip => {
+     console.log(trip)
+     $('.user-pending-trips').append(
+       `<div class='users-trips'>
+         <p>User: ${trip.name}</p>
+         <p>Trip Date: ${trip.tripDate}</p>
+         <p>Trip Status: ${trip.status}</p>
+         <button id='${trip.tripID}'>Approve Trip</button>
+         <button id='${trip.tripID}'>Denny Trip</button>
+       </div>
+       `)
```

Comment:
I don't think you need a return statement here, just running the forEach should work